doc/telekasten.txt

```diff
@@ -670,13 +670,15 @@ telekasten.toggle_todo({opts})~
   to avoid neovim sending the wrong visual selection.
 
   Valid keys for {opts}
-
+ bv  z
```

Comment:
what is that?

Reply:
@alex-laycalvert what is this `bv z`? Is it a typo?

Reply:
Yes, fixed now